nltk/tokenize/stanford.py

```diff
@@ -43,7 +43,11 @@ def __init__(self, path_to_jar=None, encoding='utf8', options=None, verbose=Fals
             searchpath=(), url=_stanford_url,
             verbose=verbose
         )
-
+        
+        # Adding logging jar files to classpath 
+        stanford_dir = self._stanford_jar.rpartition('/')[0]
```

Comment:
Use `os.path.dirname(self._stanford_jar)[0]` instead of self._stanford_jar.rpartition('/')[0] instead. That will return the parent path regardless of OS, using `/` is problematic for Windows users. My quick hack on the [gist](https://gist.github.com/alvations/e1df0ba227e542955a8a) was made within little thoughts on library maintenance =)